packages/graphile-build-pg/src/plugins/PgOrderAllColumnsPlugin.js

```diff
@@ -33,6 +34,7 @@ export default (function PgOrderAllColumnsPlugin(builder) {
               value: {
                 alias: ascFieldName.toLowerCase(),
                 specs: [[attr.name, true]],
+                unique,
```

Comment:
This is the fix; it only works for single-column unique indexes. We should add a solution for multi-column unique indexes at some point 👍

Reply:
The same change should be made to the desc enum, correct?

Reply:
Oops!

Reply:
https://github.com/graphile/graphile-engine/pull/364